src/Lorentz/Contracts/BaseDAO/FA2.hs

```diff
@@ -5,29 +5,257 @@ module Lorentz.Contracts.BaseDAO.FA2
    ( transfer
    , balanceOf
    , tokenMetadataRegistry
-   , permissionsDescriptor
    , updateOperators
+   , defaultPermissionsDescriptor
    ) where
 
 import Lorentz
 import Lorentz.Contracts.BaseDAO.Types
 import Lorentz.Contracts.Spec.FA2Interface
 
--- | TODO: Implement missing methods
+defaultPermissionsDescriptor :: PermissionsDescriptor
+defaultPermissionsDescriptor = PermissionsDescriptor
+  { pdOperator = OwnerOrOperatorTransfer
+  , pdReceiver = OwnerNoHook
+  , pdSender = OwnerNoHook
+  , pdCustom = Nothing
+  }
+
 transfer :: Entrypoint TransferParams Storage
-transfer = doNothing
+transfer = do
+  iter transferItem
+  nil; pair
+  where
+    transferItem :: forall s. TransferItem & Storage & s :-> Storage & s
+    transferItem = do
+      swap
+      getField #sAdmin
+      Lorentz.sender
+      if IsEq
+      then push True
+      else push False
```

Comment:
I suppose we can just use `eq` here. It should push the result to stack automatically.

Reply:
Changed